packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -3073,7 +3073,14 @@ const schema: SchemaType<"Posts"> = {
     label: "stale-while-revalidate caching enabled",
     group: formGroups.adminOptions,
     ...schemaDefaultValue(false),
-  }
+  },
+  generateJargon: {
+    type: Boolean,
+    optional: true,
+    nullable: true,
```

Comment:
`schemaDefaultValue` provides `nullable: false`, so this is explicitly overriden and should be removed